src/components/editor/AppController.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { browserHistory } from 'react-router';
+import FalcorController from "lib/falcor/FalcorController";
+import { setAppReady } from "lib/falcor/falcorUtils";
+
+export default class AppController extends FalcorController {
+  constructor(props) {
+    super(props);
+    this.safeSetState({
+      defaultValue: this.handleDefaultValue.bind(this)()
+    })
+    
+  }
+  static getFalcorPath() {
```

Comment:
Okay I would say it should just redirect to /articles automatically then is probably good behavior. No point keeping states like / that don't do anything.